tests/models/test_main.py

```diff
@@ -672,7 +672,9 @@ def test_brief_serialize_keys_match_api_stub_keys(self):
         brief_stub = BriefStub(
             framework_slug=self.framework.slug, status='open', lot=self.lot, clarification_questions_closed=True,
         )
-        assert sorted(brief.serialize(with_users=True).keys()) == sorted(brief_stub.response().keys())
+        assert sorted(brief.serialize().keys()) == sorted(brief_stub.response().keys())
+        assert sorted(brief.serialize(with_users=True, with_clarification_questions=True).keys()) == \
+            sorted(brief_stub.single_result_response()['briefs'].keys())
```

Comment:
Can use parenthesis implied line continuation rather than backslash
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

```
        assert(
            sorted(brief.serialize(with_users=True, with_clarification_questions=True).keys()) ==
            sorted(brief_stub.single_result_response()['briefs'].keys())
        )
```

Reply:
No problem - will fixup